pkg/utils/pod/scheduling.go

```diff
@@ -25,6 +25,13 @@ import (
 	"sigs.k8s.io/karpenter/pkg/scheduling"
 )
 
+func CanTriggerProvisioning(pod *v1.Pod) bool {
+	return !IsPreempting(pod) &&
+		FailedToSchedule(pod) &&
```

Comment:
Doesn't failed to schedule imply that the pod was marked as pending? Are you going to catch the pods that you want here? Also, do you still need this change?

Reply:
Sorry I forgot to press save!